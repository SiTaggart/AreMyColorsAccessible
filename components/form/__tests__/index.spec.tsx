/* eslint-env jest */
/// <reference types="jest" />

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Form } from '..';

describe('Form', (): void => {
  it('renders without crashing', (): void => {
    ReactDOM.render(<Form>children</Form>, document.createElement('div'));
    const formCmp = renderer.create(<Form>children</Form>).toJSON();
    expect(formCmp).toMatchSnapshot();
  });
  it('should render inline styles when past', (): void => {
    const formCmp = renderer
      .create(<Form style={{ backgroundColor: '#fff' }}>children</Form>)
      .toJSON();
    expect(formCmp).toMatchSnapshot();
  });
});
