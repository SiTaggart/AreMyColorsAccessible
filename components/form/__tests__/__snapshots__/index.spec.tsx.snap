// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders without crashing 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin: 0 auto;
  max-width: 1200px;
  padding: 0 1rem;
  -webkit-transition: color 400ms ease-out;
  transition: color 400ms ease-out;
}

<div
  className="emotion-0 emotion-1"
>
  children
</div>
`;

exports[`Form should render inline styles when past 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin: 0 auto;
  max-width: 1200px;
  padding: 0 1rem;
  -webkit-transition: color 400ms ease-out;
  transition: color 400ms ease-out;
}

<div
  className="emotion-0 emotion-1"
  style={
    Object {
      "backgroundColor": "#fff",
    }
  }
>
  children
</div>
`;
